evento/src/views/MainPage/index.js

```diff
@@ -14,12 +14,18 @@ const tabs = [
 ];
 
 class MainPage extends Component {
+	onPathChange(path) {
+		this.props.history.push(path)
+	}
 
 	render() {
 		return (
 			<div className="MainPage">
 				<SearchBar />
-				<TabContainer tabs={tabs} />
+				<TabContainer
+					tabs={tabs}
+					path={this.props.location.pathname}
+					onPathChange={(path) => this.onPathChange(path)}/>
```

Comment:
It makes no sense that TabContainer has a "onPathChange" event. Should be onSelectedChange or onSelectedTabChange